app/editor/src/features/admin/filters/utils/generateMultiMatch.ts

```diff
@@ -5,22 +5,19 @@
  * @returns An Elasticsearch query for either a multi_match or a match.
  */
 export const generateMultiMatch = (fields: string | string[], values: any) => {
+  if (values === undefined || values === null) return undefined;
```

Comment:
All the functions return `undefined` when there isn't anything to return.  This makes more sense and is easier to implement.